src/api/goods.ts

```diff
@@ -3,17 +3,29 @@ import { Good } from '../types/Good';
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export function getAll(): Promise<Good[] | null> {
```

Comment:
The type of the Promise returned by the getAll function has been changed from Good[] to Good[] | null. This might not be necessary as fetch API doesn't return null but throws an error in case of failure.

```
return fetch(API_URL)
    .then(response => {
      if (!response.ok) {
        throw new Error(`${response.status} ${response.statusText}`);
      }

      return response.json();
    })
```